conf/app.routes

```diff
@@ -10,3 +10,59 @@ GET        /complete-declaration/more-information/:departureId                co
 GET        /complete-declaration/check-information/:departureId               controllers.CheckInformationController.onPageLoad(departureId: String)
 
 GET        /:departureId                                                      controllers.IndexController.index(departureId: String)
+
```

Comment:
Hello Ken, the structure for this has changed and the routes that seat in the location-of-goods section has been moved out of this file.